editor/js/Sidebar.Material.BooleanProperty.js

```diff
@@ -12,24 +12,28 @@ function SidebarMaterialBooleanProperty( editor, property, name ) {
 	container.add( boolean );
 
 	let object = null;
+	let materialSlot = null;
 	let material = null;
 
 	function onChange() {
 
 		if ( material[ property ] !== boolean.getValue() ) {
 
-			editor.execute( new SetMaterialValueCommand( editor, object, property, boolean.getValue(), 0 /* TODO: currentMaterialSlot */ ) );
+			editor.execute( new SetMaterialValueCommand( editor, object, property, boolean.getValue(), materialSlot ) );
 
 		}
 
 	}
 
-	function update() {
+	function update( currentObject, currentMaterialSlot = 0 ) {
+
+		object = currentObject;
+		materialSlot = currentMaterialSlot;
 
 		if ( object === null ) return;
 		if ( object.material === undefined ) return;
```

Comment:
Is this a correct check if now different expression is assigned to material? /ping @Mugen87